### PR TITLE
proto: get transaction input accepts only one account, fixes #103

### LINKED ISSUE #103: `GetTransactionInputRequest` should only take 1 account id
See https://github.com/0xPolygonMiden/miden-node/pull/85#discussion_r1423303321

### PR DESCRIPTION
**Pull request by hackaugusto**
Closes #103

As the PR title says.

### CODE REVIEW

**Comment by hackaugusto** [comment]
tests are failing for unrelated reason, #109 fixes them

**Review by plafer — APPROVED** [review]
LGTM (after we merge #109 and confirm CI passes)

**Pull request merged**
